Yoast/developer-site
# Introduced a plugin to allow us to load in assets from the docs

This PR allows us to use images needed in the docs, without needing to ship it in this repository.
Essentially, this means we can just ship images in the docs repository, reference them in the docs and upon a successful build, said images will be properly mapped into the `img` directory.

To test:
* Checkout this PR
* Checkout https://github.com/Yoast/developer-docs/pull/37
* Run `yarn build`
* Check the `/build/img/` dir and see that images shipped in the docs repository, are present.
